docs/user/mobile-applications.rst

```diff
@@ -93,13 +93,119 @@ Problems
 
 Please report any issues on the `GitHub repository <https://github.com/ls1intum/artemis-android/issues>`_.
 
-.. |server-selection-overview| image:: native-applications/android/server_selection_overview.png
+iOS Application
+---------------
+
+The iOS application supports the following features:
+
+#. Login:
+    #. Select predefined and custom Artemis servers
+    #. Login with username + password
+
+.. raw:: html
+
+    <iframe src="https://live.rbg.tum.de/w/artemisintro/35200?video_only=1&t=0" allowfullscreen="1" frameborder="0" width="600" height="350">
```

Comment:
The iframe does not load for me in Safari and Firefox, can you render a fallback link?